subprojects/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java

```diff
@@ -199,6 +206,10 @@ public void setupProjectLayout() {
 
         validatePackageName(packageName);
 
+        java.util.Optional<JavaLanguageVersion> toolChainVersion = getJavaLanguageVersion(useIncubatingAPIs, inputHandler, initDescriptor.getLanguage());
```

Comment:
I think all of the other options that can be prompted for also have a command-line equivalent. This option is similar to the testing framework, but even more specific to the Java ecosystem. 

The proper way of doing this would be to have a generic "target" option that could be reused for any ecosystem starting with the JVM, but that looks like more work to figure out how to do that in a nice way.

For now, I think I would not have a CLI option until someone complains.